tests/verify-fallback-package-v2/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "moduleResolution": "Node",
+    "target": "es2020"
```

Comment:
Split out the verifications for v1 and v2. We care that v1 works with es2018. es2020 is needed for v2 for type-fest to resolve correctly. So we separate the test into two.